### PR TITLE
P.A.C.M.A.N. Tweak: Phoron Scarcity Edition

### PR DESCRIPTION
**Pull request by furrycactus**
Slightly reworked the P.A.C.M.A.N. portable generators to be cooler with the Phoron scarcity.

The Phoron generator produces the most power out of all three of them now, but has the lowest fuel efficiency, and has catastrophic consequences if it overheats (a bigger explosion, phoron gas, radiation, oh my).

The Tritium generator produces less power than the Phoron one, but still makes a considerable amount of power, considering it runs off of an isotope of Hydrogen which is used for _nuclear fusion_. It's not radioactive however, but the explosion size is quite large if it overheats.

The Uranium generator produces the least power out of all three, but about the same amount as the Phoron one used to before this change (it already did this, I didn't change the value). It has incredibly good fuel efficiency as well and can create power for a very long time, with the downside of producing very tiny amounts of radiation when active, and also delivering a heavy burst of radiation should it overheat and explode.

The Phoron fueled versions have been largely phased-out on the Aurora, being replaced with the older Uranium and Tritium fueled technology. As such, Engineering doesn't have Phoron sheets anymore, instead they have a small supply of Uranium and Tritium. 

The P.A.C.M.A.N.s in Telecoms have been swapped out with Tritium fueled variants. The one in the Command Bunker has been swapped out with a Uranium fueled variant.

### CODE REVIEW

**Comment by furrycactus** [comment]
!review

**Comment by furrycactus** [comment]
Preferably merge with or after #10547

File: code/modules/power/port_gen.dm [hunk 1]
```diff
@@ -90,10 +90,10 @@
 #define TEMPERATURE_DIVISOR 40
 #define TEMPERATURE_CHANGE_MAX 20
 
-//A power generator that runs on solid plasma sheets.
+//A power generator that runs on solid phoron sheets.
 /obj/machinery/power/port_gen/pacman
-	name = "\improper P.A.C.M.A.N.-type Portable Generator"
-	desc = "A power generator that runs on solid phoron sheets. Rated for 80 kW max safe output."
+	name = "\improper P-P.A.C.M.A.N.-type Portable Generator"
+	desc = "An advanced power generator that runs on solid phoron sheets. Rated for 120 kW max safe output. WARNING: DO NOT OPERATE ABOVE SAFE THRESHOLD FOR EXTENDED PERIODS."
```
Comment (NonQueueingMatt) [inline code/modules/power/port_gen.dm:96]: For extra epic factor:
```suggestion
	desc = "An advanced power generator that runs on solid phoron sheets. Rated for 120 kW max safe output. <span class='danger'>WARNING: DO NOT OPERATE ABOVE SAFE THRESHOLD FOR EXTENDED PERIODS.</span>"
```

File: code/modules/power/port_gen.dm [hunk 2]
```diff
@@ -254,6 +254,13 @@
 	var/datum/gas_mixture/environment = loc.return_air()
 	if (environment)
 		environment.adjust_gas_temp(GAS_PHORON, phoron/10, temperature + T0C)
+	var/rads = 90 + (sheets + sheet_left)*1.5
+	for (var/mob/living/L in range(src, 10))
+		//should really fall with the square of the distance, but that makes the rads value drop too fast
+		//I dunno, maybe physics works different when you live in 2D -- SM radiation also works like this, apparently
+		L.apply_effect(max(20, round(rads/get_dist(L,src))), IRRADIATE, blocked = L.getarmor(null, "rad"))
+	explosion(loc, 3, 6, 12, 16, 1) //explosion about as big as a Supermatter shard, also produces radiation. Mars told us phoron is dangerous.
```
Comment (NonQueueingMatt) [inline code/modules/power/port_gen.dm:262]: Does this give an appropriate warning before violently Marsing the atmosphere? I think it should. Sirens and all.

File: code/modules/power/port_gen.dm [hunk 3]
```diff
@@ -386,12 +393,13 @@
 		SSvueui.check_uis_for_change(src)
 
 /obj/machinery/power/port_gen/pacman/super
-	name = "S.U.P.E.R.P.A.C.M.A.N.-type Portable Generator"
-	desc = "A power generator that utilizes uranium sheets as fuel. Can run for much longer than the standard PACMAN type generators. Rated for 80 kW max safe output."
+	name = "U-P.A.C.M.A.N.-type Portable Generator"
+	desc = "A power generator that utilizes uranium sheets as fuel. Can run for much longer than the standard PACMAN type generators. Rated for 80 kW max safe output. WARNING: MINOR RADIATION HAZARD WHEN ACTIVE. DO NOT OPERATE ABOVE SAFE THRESHOLD FOR EXTENDED PERIODS."
```
Comment (NonQueueingMatt) [inline code/modules/power/port_gen.dm:397]:
```suggestion
	desc = "A power generator that utilizes uranium sheets as fuel. Can run for much longer than the standard PACMAN type generators. Rated for 80 kW max safe output. <span class='warning'>WARNING: MINOR RADIATION HAZARD WHEN ACTIVE. DO NOT OPERATE ABOVE SAFE THRESHOLD FOR EXTENDED PERIODS.</span>"
```

File: code/modules/power/port_gen.dm (hunk 2 above)
Comment (furrycactus) [reply code/modules/power/port_gen.dm:263]: I will do this once I work out how to.

**Comment by NonQueueingMatt** [comment]
Reopen when ready.

**Pull request closed without merging**